routers/repo/pull.go

```diff
@@ -351,7 +351,7 @@ func PrepareViewPullInfo(ctx *context.Context, issue *models.Issue) *git.Compare
 	ctx.Data["Divergence"] = divergence
 	allowUpdate, err := pull_service.IsUserAllowedToUpdate(pull, ctx.User)
 	if err != nil {
-		ctx.ServerError("GetDiverging", err)
+		ctx.ServerError("IsUserAllowedToUpdate", err)
```

Comment:
I think this was an accidental change?

Reply:
The change itself makes sense, just not I'm this particular PR.

Reply:
@jolheiser I know, but this is noice witch I dont like to create a extra PR for one line :/
was from: #9784 ... -> [link](https://github.com/go-gitea/gitea/pull/9784#discussion_r367764502)

Reply:
Yeah, fair enough. At the very least mention it in the PR description.  
It's never going to matter (probably) but just nice to know it was intentional.